cmd/commands/service/serviceTypes_linux.go

```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/mysteriumnetwork/node/core/service"
 	service_noop "github.com/mysteriumnetwork/node/services/noop"
 	service_openvpn "github.com/mysteriumnetwork/node/services/openvpn"
-	service_wireguard "github.com/mysteriumnetwork/node/services/wireguard"
+	"github.com/mysteriumnetwork/node/services/wireguard/wgdefinition"
```

Comment:
I see that it was required specifically for mobile, but was so nice to have the same appearance for all wireguard, noop and openvpn. Maybe we can do something similar for all packages in the future? 

And `wgdefinition` looks strange. At least it stutters `wireguard/wgdefinition`.

Reply:
Yea package importing is a bit cumbersome. Mayb wg definition could lay somewhere on the side. Or maybe we can have a separate package with all service definitions registered not - sure. The problem was that wireguard package is so fat with unrelated code that even to register service definition json parse (simple function no wg specifics) requires all wg package to be build with all dependencies (wgctrl!) which does not even compile to armv7 as we already know. I have no clear way to move here - any suggestions would be fine. What I really don't like is that consumer code needs to import whole package with provider code just to register unserializer.

Reply:
I think we should definitely move the json bootstrap part away from all the services. Maybe put it beside the rest of the package "service_discovery"(which on its own needs a reconsideration). That way we can register the relevant serializers from that package, instead of having each service define their own?

Reply:
refactored by @soffokl